book/website/collaboration/inclusive-events/inclusive-events-comms.md

```diff
@@ -0,0 +1,487 @@
+
+In this announcement, you should provide information about any specific accessibility and inclusion requirements that are addressed such as whether things like captions, interpretation, or other accessibility practices will be available. 
+
+If possible, consider establishing a point of contact on your team who is able to address any questions or concerns that people with lived experience may have about attending your event. 
+This may involve setting up an email inbox such as accessibility@yourevent.com. 
+
+By providing clear and proactive information about how your event addresses barriers to inclusion, people with lived experience will be able to find out whether the event will be accessible to them individually, and decide if they are able to attend. 
+For example, communicating that your event will be hybrid may allow people who otherwise would not engage in the event to attend online. 
+
+This announcement should also contain information about how to register for the event, the finalized location, and a preliminary program. 
+
+```{tip}
+Autism Advocacy suggests that schedules for multi-day events should be sent at least one month in advance which includes details about transport
+and its proximity to the event location, training session names, speaker names, breaks, and planned schedule, and at least 2 weeks in advance for events taking place on a single day.
+The [Autism Advocacy Accessible Event Planning Guide](https://autisticadvocacy.org/wp-content/uploads/2019/05/Accessible-Event-Planning.pdf) also contains other recommendations and timeline guidance for running events for autistic people.
+```
+
+
+(cl-inclusive-events-comms-website)=
+## Using an Event Website 
+It can be incredibly helpful to centralize information about your event in both online and offline locations.
+
+Consider if your event will need a website and dedicate a page on your site to information and a point of contact regarding accessibility requirements and the steps you are taking to make the event inclusive for people with lived conditions and accessibility requirements. 
+
+You can also choose an event platform such as Eventbrite, Humanitix, or EventYay to host information about your event. 
+When deciding on what platform to use please review the accessibility of a potential site, and follow any advice as to how you can make your content on their platform accessible. 
+For example, [Eventbrite provides accessibility guidance](https://www.eventbrite.com/blog/66754-2how-to-creat-an-accessible-event-listing-ds00/) for event pages hosted on their platform. 
+
+There are two main considerations for your event website:
+{ref}`cl-inclusive-events-comms-website-accessibility` and {ref}`cl-inclusive-events-comms-website-content`. 
+
+(cl-inclusive-events-comms-website-accessibility)=
+### Website Accessibility
+Your website should meet Web Accessibility Standards by following the 4 WCAG principles: 
```

Comment:
What is WCAG? :D

Reply:
Beyond that, this section might potentially be a bit too in-depth here given that the focus is on events broadly and not so much on web-design?

Reply:
Updated with the full acronym - and then coming back again later when I start pulling out sections.